src/client/entity-editor/edition-section/actions.ts

```diff
@@ -95,7 +93,7 @@ export function updateFormat(newFormatId: ?number): Action {
  *                 date.
  * @returns {Action} The resulting UPDATE_RELEASE_DATE action.
  */
-export function debouncedUpdateReleaseDate(newReleaseDate: ?object): Action {
+export function debouncedUpdateReleaseDate(newReleaseDate: number | null | undefined): Action {
```

Comment:
this was `?object`, which doesn't help much determine the structure of the argument, but I think transposing it to `number` is a typo.

After looking at the code again, it should be  `string | null` (the DateField component that uses this action uses the `dateObjectToISOString` method under the hood)

Reply:
Agreed. I think this slipped in because we have no type information on `common/new-date-field.js`, so TypeScript didn't have any type information for `DateField.onChangeDate`. Will change.